SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -1,30 +1,221 @@
+        case .photo:
+            PersonPhotoTableViewCell.register(to: tableView, identifier: identifier)
+        case .info:
+            PersonInfoTableViewCell.register(to: tableView, identifier: identifier)
+        case .story:
+            PersonOverviewTableViewCell.register(to: tableView, identifier: identifier)
+        case .outcome:
+            PersonOverviewTableViewCell.register(to: tableView, identifier: identifier)
+        case .news:
+            PersonNewsTableViewCell.register(to: tableView, identifier: identifier)
+        case .medias:
+            PersonNewsTableViewCell.register(to: tableView, identifier: identifier)
+        }
+    }
+    
+    static var allCases: [PersonCellType] {
+        return [.photo, .info, .story, .outcome, .news([]), .medias([])]
+    }
+    
+    static func == (lhs: PersonCellType, rhs: PersonCellType) -> Bool {
+        return lhs.identifier == rhs.identifier
+    }
+}
 
+class PersonController: BaseViewController {
+    
+    var person: Person!
+    
+    var sareArea: UILayoutGuide!
+    var tableView: UITableView = UITableView(frame: .zero)
+    
+    let donationButtonContainerView = ButtonContainerView(frame: .zero)
+    
+    var cellCollectionTypes: [PersonCellType] = {
+        return [.photo, .info, .story, .outcome, .news([]), .medias([])]
+    }()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         view.accessibilityIdentifier = "personView"
+    }
+    
+    override func loadView() {
+        super.loadView()
+        sareArea = view.layoutMarginsGuide
+        view.backgroundColor = .white
+        setupNavigationBarItems()
+        setupTableView()
+        setupDonationButton()
+    }
+    
+    private func setupNavigationBarItems() {
+        
+        let dismissActionGesture = UITapGestureRecognizer(target: self, action: #selector(dismissAction(_:)))
+        let shareActionGesture = UITapGestureRecognizer(target: self, action: #selector(shareAction(_:)))
 
-        // Do any additional setup after loading the view.
+        let dismissButton = UIButton(type: .system)
+        dismissButton.addGestureRecognizer(dismissActionGesture)
+        dismissButton.setImage(UIImage(named: "Close Icon")?.withRenderingMode(.alwaysOriginal), for: .normal)
+        dismissButton.frame = CGRect(x: 0, y: 0, width: 34, height: 34)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: dismissButton)
+        
+        let shareButton = UIButton(type: .system)
+        shareButton.addGestureRecognizer(shareActionGesture)
+        shareButton.setImage(UIImage(named: "share_white")?.withRenderingMode(.alwaysOriginal), for: .normal)
+        shareButton.frame = CGRect(x: 0, y: 0, width: 34, height: 34)
+        navigationItem.rightBarButtonItem = UIBarButtonItem(customView: shareButton)
+        
+        navigationController?.navigationBar.isTranslucent = false
+        navigationController?.navigationBar.barTintColor = .black
+    }
```

Comment:
Think it should be useful if we put this setting into an extension for later reuse in other view controllers.